imports/ui/case/chatbot-ui.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react'
+import moment from 'moment'
+import styles from './case.mss'
+import { Link } from 'react-router-dom'
+import { UneeTIcon } from '../components/unee-t-icons'
+import IconButton from 'material-ui/IconButton'
+import {
+  logoIconStyle,
+  logoButtonStyle
+} from '../components/app-bar.mui-styles'
+
+export default class ChatBotUI extends Component {
+  assigneeBot () {
+    const creationTime = (new Date()).toISOString()
+    const assignedUserRole = this.props.caseItem.assignedUnitRole
+    return (
+      <div>
+          You've created a new case! The case has been assigned to <span className='fw5'> {assignedUserRole} </span>and we've notified them.
+        <div className={styles.messageTime + ' fr f7'}>
+          {moment(creationTime).format('HH:mm')}
+        </div>
+      </div>
+    )
+  }
+
+  inviteBot () {
+    const caseId = this.props.caseItem._id
+    return (
+      <Link to={`/case/${caseId}/details/invite`} className='link'>
+        <div className='tc bondi-blue pv2'>
+        Invite People
+        </div>
+      </Link>
+    )
+  }
+
+  photoBot () {
+    return (
+      <div className='tc bondi-blue pv2'>
+        <label>
+          Add Photo
+          <input type='file' className='dn' onChange={this.props.handleFileSelection.bind(this)} />
+        </label>
+      </div>
+    )
+  }
+
+  render () {
+    const isCurrentUserCreator = this.props.caseItem.creator === this.props.userBzLogin
+    const chatbotMsg = [
+      {id: 1, msg: this.assigneeBot(), link: null},
+      {id: 2, msg: 'Would you like to inform someone else about this case?', link: this.inviteBot()},
+      {id: 3,
+        msg: <span><span className='fw5'>Tip: </span> You can also add photos to your case so your correspondents have a better understanding of the case</span>,
+        link: this.photoBot()}
+    ]
+    return (
+      <div>
+        { isCurrentUserCreator &&
```

Comment:
There's actually no point in rendering this entire component if the user is not the owner. This check and concern should be moved up to the parent component,